templates/components/pager.html.twig

```diff
@@ -66,62 +66,65 @@
 {% set skip_adjacents = false %}
 
 <div class="flex-grow-1 d-flex flex-wrap flex-md-nowrap  align-items-center justify-content-between mb-2 search-pager">
-   {% set limitdropdown = include('components/dropdown/limit.html.twig', {
-      'no_onchange': true,
-      'select_class': 'search-limit-dropdown',
-   }) %}
-   <span class="search-limit d-none d-md-block">
-      {{ __('%s rows / page')|format(limitdropdown)|raw }}
-   </span>
-   <span class="search-limit d-block d-md-none">
-      {{ limitdropdown|raw }}
-   </span>
-   <p class="m-0 text-muted d-none d-md-block page-infos">
-      {{ __('Showing %s to %s of %s rows')|format(current_start, current_end, count) }}
-   </p>
-   <p class="m-0 text-muted d-block d-md-none text-nowrap ms-2 page-infos">
-      {{ __('%s-%s/%s')|format(current_start, current_end, count) }}
-   </p>
-   <ul class="pagination m-0 mt-sm-2 mt-md-0">
-      {% set is_first_page = start == 0 %}
-      {% set is_last_page = forward >= count %}
+    {% set limitdropdown = include('components/dropdown/limit.html.twig', {
+        'no_onchange': true,
+        'select_class': 'search-limit-dropdown',
+    }) %}
+    <span class="search-limit d-none d-md-block">
+        {{ __('%s rows / page')|format(limitdropdown)|raw }}
+    </span>
+    <span class="search-limit d-block d-md-none">
+        {{ limitdropdown|raw }}
+    </span>
+    <p class="m-0 text-muted d-none d-md-block page-infos">
+        {{ __('Showing %s to %s of %s rows')|format(current_start, current_end, count) }}
+    </p>
+    <p class="m-0 text-muted d-block d-md-none text-nowrap ms-2 page-infos">
+        {{ __('%s-%s/%s')|format(current_start, current_end, count) }}
+    </p>
 
-      <li class="page-item {% if is_first_page %}disabled{% endif %}">
-         <a class="page-link page-link-start" href="{{ href|replace({'%start%': 0}) }}" title="{{ __('Start') }}" data-start="0" {% if is_first_page %}aria-disabled="true"{% endif %}>
-            <i class="ti ti-chevrons-left"></i>
-         </a>
-      </li>
-      <li class="page-item {% if is_first_page %}disabled{% endif %}">
-         <a class="page-link page-link-prev" href="{{ href|replace({'%start%': back}) }}" title="{{ __('Previous') }}" data-start="{{ back }}" {% if is_first_page %}aria-disabled="true"{% endif %}>
-            <i class="ti ti-chevron-left"></i>
-         </a>
-      </li>
-      {% for page in range(1, nb_pages) %}
-         {% if current_page - adjacents <= page and current_page + adjacents >= page %}
-            {% set page_start = (page - 1) * limit %}
-            <li class="d-none d-sm-block page-item {% if page == current_page %}active selected{% endif %}">
-               <a class="page-link page-link-num" href="{{ href|replace({'%start%': page_start}) }}" data-start="{{ page_start }}">{{ page }}</a>
+    <ul class="pagination m-0 mt-sm-2 mt-md-0">
```

Comment:
Is displaying this empty `ul` necessary ?

Reply:
yes it's done on purpose, with this, `justify-content-between` on parent still work corrrectly and display the "Showing x to x of x rows" on middle of the div